### ISSUE TITLE
ESPN changed their college football format.

### ISSUE THREAD

**Issue by Rumel** [swanpenguin/Reddit-Post-Game-Thread-Generator#6]
![screen shot 2015-08-30 at 4 28 16 pm](https://cloud.githubusercontent.com/assets/1217404/9569542/28fad31c-4f34-11e5-9af3-a6f6cb82fce4.png)

![screen shot 2015-08-30 at 4 28 59 pm](https://cloud.githubusercontent.com/assets/1217404/9569547/4bc83eb6-4f34-11e5-992e-1bfae9eeb153.png)

**Comment by Rumel** [comment]
@swanpenguin We should probably make a separate generator for each of the sports. We could move to a structure like this:

```
/lib
  /espn_box_score
    ncf.rb
    nba.rb
    ncb.rb
    ncw.rb
  espn_box_score.rb
```

`espn_box_score.rb` would do a url match and then hand off to the generator and return. We could also add a utilities file for shared code. I think we might be able to write some helper functions to dry up the xpath code.

**Comment by swanpenguin** [comment]
@Rumel I agree with you. That would be a good model. I think we should adopt it. Further, I got rid of the **Top Performers** portion of the created post. It can be replaced easily with another part of the data. There is some data when you use `/game?` instead of `/boxscore?` in the url.

It would be awesome if we could get some gifs or vids of highlights of the game, but... would probably need another data source.
